config/schema/schema.config

```diff
@@ -211,6 +217,10 @@ schema {
             } else if (properties.containsKey('choices')) {
                 schema.check_choices_singular(options, name, properties.choices)
             }
```

Comment:
**question (non-blocking):** Do we still need this check? With your additional check added below outside of this if-else tree, it seems like this will unnecessarily run `schema.check_choices_singular` again for `Number` and `Integer` types.

Reply:
That's totally fair, I was avoiding removing it; removed it with [#b52420d](https://github.com/uclahs-cds/pipeline-Nextflow-config/pull/71/commits/b52420d88b069c77dd1b1dea908879230239394f)